src/Markdig/Renderers/Normalize/ListRenderer.cs

```diff
@@ -37,7 +37,7 @@ protected override void Write(NormalizeRenderer renderer, ListBlock listBlock)
                     renderer.Write(index + "");
                     renderer.Write(listBlock.OrderedDelimiter);
                     renderer.Write(' ');
-                    renderer.PushIndent("  ");  // TODO: output accurate indent
+                    renderer.PushIndent(new string(' ', IntLog10Fast(index) + 3));
```

Comment:
Not sure to understand the logic here. Why a higher index would generate an indent?

Reply:
because "10." has less characters than "123456."

Reply:
Ah ok, so you want to left pad with spaces.. isn't something that should be better part of a normalizing stuffs and not the default one? (Not sure I align usually list items by left padding with spaces...)

Reply:
![image](https://user-images.githubusercontent.com/442577/31951667-111ad942-b8df-11e7-8844-2f75043c6b7e.png)

Reply:
han ok sorry, a bit slow today 😅